doc/getting-started/overview_dev.rst

```diff
@@ -158,101 +167,123 @@ about the target hardware and generate a board configuration file.
 .. |icon_target| image:: ./images/icon_target.png
    :scale: 75%
 
-A **board configuration file** is an XML file that stores hardware-specific
-information extracted from the target system. It describes the capacity of
-hardware resources (such as processors and memory), platform power states,
-available devices, and BIOS settings. The file is used to configure the ACRN
+The :ref:`board_inspector_tool` ``board_inspector.py`` enables you to generate a
+board configuration file on the target system.
+
+A **board configuration file** stores hardware-specific information extracted
+from the target system. This XML file describes the capacity of hardware
+resources (such as processors and memory), platform power states, available
+devices, and BIOS settings. The file is used to configure and build the ACRN
 hypervisor, because each hypervisor instance is specific to your target
 hardware.
 
-The **board inspector tool** ``board_inspector.py`` enables you to generate a board
-configuration file on the target system. The following sections provide an
-overview and important information to keep in mind when using the tool.
+The following sections provide an overview and important information to keep
+in mind when using the Board Inspector.
 
 Configure BIOS Settings
 =======================
 
-You must configure all of your target's BIOS settings before running the board
-inspector tool, because the tool records the current BIOS settings in the board
+You must configure all of your target's BIOS settings before running the Board
+Inspector tool, because the tool records the current BIOS settings in the board
 configuration file.
 
-Some BIOS settings are required by ACRN. The :ref:`gsg` provides a list of the
-settings.
+ACRN requires the following BIOS settings:
```

Comment:
There's also a third BIOS setting (disable secure boot).  I'm not sure you want to duplicate these settings from the GSG here anyway (means we'd need to keep two places up to date).